test/e2e/suite/command/verify.go

```diff
@@ -240,7 +240,7 @@ var _ = Describe("notation verify", func() {
 			notation.Exec("sign", "--timestamp-url", "http://timestamp.digicert.com", "--timestamp-root-cert", filepath.Join(NotationE2EConfigPath, "timestamp", "DigiCertTSARootSHA384.cer"), artifact.ReferenceWithDigest()).
 				MatchKeyWords(SignSuccessfully)
 
-			notation.Exec("verify", artifact.ReferenceWithDigest(), "-v").
```

Comment:
What's this change for?

Reply:
This is due to notation-go log update. Some logs are moved from `INFO` to `DEBUG` level, hence `-v` will no longer show the expected log message.